stages/org.osbuild.anaconda.interactive-defaults

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Create an Anaconda interactive defaults file
+
+Interactive defaults files are a way to program
+the Anaconda installer.  This stage can be used
+to create such an interactive defaults file at
+the location expected by anaconda.
+
+Only a very limited set of available kickstart
+commands are supported here.
+"""
+
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "liveimg": {
+    "type": "object",
+    "required": ["url"],
+    "properties": {
+      "url": {
+        "type": "string"
+      }
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    path = "usr/share/anaconda/interactive-defaults.ks"
+
+    config = []
+
+    liveimg = options.get("liveimg")
+    if liveimg:
+        url = liveimg["url"]
+        config += [f"liveimg --url {url}"]
+
+    target = os.path.join(tree, path)
+    base = os.path.dirname(target)
+    os.makedirs(base, exist_ok=True)
+
+    with open(target, "w", encoding="utf8") as f:
```

Comment:
```suggestion
    with open(target, "x", encoding="utf8") as f:
```

Or do you explicitly want to override existing values?

Reply:
I do want to explicitly override existing values; it'd be too easy to add conflicting information to the interactive-defaults.ks otherwise.

Reply:
I was not talking about amending it, I was proposing to fail if it already exists.

Reply:
This file is created by `anaconda-core` and is always present.